coreModulePackages/promise/index.js

```diff
@@ -16,6 +16,6 @@
 // with Webpack 2+. We need `require('promise-polyfill').default` for running the tests
 // and `require('promise-polyfill')` for building Turbine.
 module.exports =
-  window.Promise ||
```

Comment:
This was using the native promise if it was available. Don't you want to do the same for node?

Reply:
@Aaronius I've added a check for Node.